doc/Client-Portal Protocol.md

```diff
@@ -452,6 +454,10 @@ A publication can send either media or data, but a QUIC *transport* channel can
             mid: string(MID),
             from: string(TrackID) | string(StreamID),
             parameters: object(VideoParametersSpecification) | undefined,
+            preferredLayers: { // Used to force layers for SVC stream
```

Comment:
shall we move this to parameters above? And the same shall be shared by subscription control singaling.

Reply:
My consideration is that the `preferredLayers` may have conflict with `parameters.resolution` or `parameters.framerate`. For example, spatial layer 0 and resolution 720p may not align. Placing it outside `parameters` can mean that it will override `parameters`.
Besides, I have no idea about describing the parameters of an SVC stream yet. Because an SVC stream may have multiple resolutions.